sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -162,6 +162,22 @@ private PollerFlux(Duration pollInterval,
         this.syncActivationOperation = cxt -> activationOperation.apply(cxt).block();
     }
 
+    /**
+     * Creates a poller flux instance that returns an error on subscription.
+     *
+     * @param ex The exception to be returned on subscription of this {@link PollerFlux}.
+     * @param <T> The type of poll response value.
+     * @param <U> The type of the final result of long running operation.
+     * @return A poller flux instance that returns an error without emitting any data.
+     *
+     * @see {@link Mono#error(Throwable)}
+     * @see {@link Flux#error(Throwable)}
+     */
+    public static <T, U> PollerFlux<T, U> error(Exception ex) {
```

Comment:
Do we want this to take a `ClientLogger` so the exception can be logged? Or is it expected for the caller to log this exception?

Reply:
This would follow the pattern when dealing with `Mono.error`, `Flux.error`, and `PagedFlux.error`.

Reply:
Logging is expected to be done by the caller. This is just like `Mono.error()`. 

What you are referring to is `FluxUtils.monoError()`.